pkgs/development/libraries/libwacom/default.nix

```diff
@@ -16,5 +16,6 @@ stdenv.mkDerivation rec {
     platforms = platforms.linux;
     homepage = https://sourceforge.net/projects/linuxwacom/;
     description = "Libraries, configuration, and diagnostic tools for Wacom tablets running under Linux";
+    license = licenses.gpl2Plus;
```

Comment:
The Git repo is now hosted on GitHub and the license looks like a slightly modified (includes Red Hat) MIT license: https://github.com/linuxwacom/libwacom/blob/master/COPYING

Reply:
Okay, I'll change that. Do you think its a good idea to update the library in this PR as well?

Regarding the license, [this site](https://fedoraproject.org/wiki/Licensing:MIT) has a list of MIT variants. The one used in `libwacom` seems to be some kind of mixture of some of these, especially the "old style" ones.
I think that they are probably functionally identical. Is it safe to use `licenses.mit` here?

Reply:
Yeah, I also looked at that site before claiming it looks like MIT. :smile:  I think that `licenses.mit` should be safe here. Thanks!

You can either update the library in this PR or open a new PR, doesn't matter IMHO. It should be a separate commit though. :+1:

Reply:
I think it's probably better to put the update and repository-change inside another PR. I'll change the license to be `mit`.